README.md

```diff
@@ -3,6 +3,8 @@
 Use Nextcloud as an identity provider for external services
 using the [SCIM] standard.
 
+:warning: This app is still under development.
```

Comment:
What do you think about using Github-flavored alerts instead?
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts

Reply:
Yep, done.